### ISSUE TITLE
f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#163]
**Description:**

Same as f.023 + turning off the clubb trapezoidal rules
```
clubb_l_trapezoidal_rule_zm          =  .false.
clubb_l_trapezoidal_rule_zt          =  .false.
```

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024

On github:
https://github.com/NCAR/amwg_dev/tree/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_058_MOM_e

On github:
hash: 8f70c08

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024/atm/](https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024/atm/)

**Contacts:**
@JulioTBacmeister, @adamrher  and @cecilehannay

**Comment by justin-richling** [comment]
ADF diags for 24 vs 23 are [here](https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024/atm/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024_1979_1984_vs_f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023_1979_1984/)

I left out a handful of variables to help speed up the ADF run, if there are some variables you'd like to see I can quickly add them in.

Also please ignore the case name in the titles of the baseline plots, I forgot to update the case name. The correct names are listed at the top of the pages in the header

**Comment by justin-richling** [comment]
Recent ADF run 1979-1990 [here](https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024/atm/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024_1979_1990_vs_f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023_1979_1990/)

**Comment by JulioTBacmeister** [comment]
Comparison of D(RESTOM) in Years 1-12 of B1850 and 1979-1900 of FWscHIST using 023 configuration and 024 configuration.  I see lots of geographical similarity between the changes in B-case and F-case runs.  Change is a little stronger in F-case 0.76 Wm-2 versus 0.52 in B1850.

![image](https://user-images.githubusercontent.com/9221710/186530216-142ed662-7ded-40ea-8ff1-2149d47abba6.png)

**Comment by dan800** [comment]
@JulioTBacmeister @cecilehannay A quick look indicates 024 has ~5% more low cloud than 023, reflecting more shortwave radiation and decreasing RESTOM by about 1.6W/m2. RESTOM would generally be warming the planet (if it were interactive), so the rate of warming would be less. This is assuming I am reading the labels and signs correctly. Is that your take?

**Comment by dan800** [comment]
Are the labels wrong in the plots? 
https://webext.cgd.ucar.edu/FWscHIST/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024/atm/f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.024_1979_1984_vs_f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023_1979_1984/html_img/plot_page_CLDLOW_ANN_LatLon_Mean.html
Why does it say b case?

**Comment by justin-richling** [comment]
Yeah that is a mistake, it should be f.cesm3_cam058_mom_e.FWscHIST.ne30_L58.023
